### ISSUE TITLE
0.30.0 Release Notes

### ISSUE THREAD

**Issue by caridy** [salesforce/lwc#645]
**LWC 0.30.0 Release Notes - Revert Forward and Fixes**

__WIP__

TL;DR: ...TBD...

BREAKING CHANGES

**`tabindex` attribute and `tabIndex` property can only be set to empty, `0` or `-1` . (risk is very low, but if you're using some other value, it will automatically be normalized to the proper value.

**`this.toString()` in components changed from `<x-foo>` to `[object Foo]` to match regular web components. https://github.com/salesforce/lwc/pull/643

**`this.tagName` is now disallowed, this is to prevent creating logic that depends on a tagName that you're not entitle to define. The tagName is always defined by the consumer of the component, and creating a component that attempts to use that value is considered harmful. https://github.com/salesforce/lwc/pull/643

**the `is` attribute is now disallowed.

**`forceTagName` static flag in components is now longer supported to prevent component author to dictate what tagName should be used.

FIXES

...TBD

NEW

**`slotchange` event for `<slot>` elements
